easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2021a.eb

```
@@ -0,0 +1,44 @@
+# Author: Denis Krišťák (INUITS)
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Update: Erica Bianco <erica.bianco@hpcnow.com>
+# License::     GPL-v3.0
+easyblock = "PythonBundle"
+
+name = 'umap-learn'
+version = '0.5.3'
+
+homepage = 'https://umap-learn.readthedocs.io/en/latest/'
+description = """Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique 
+                 that can be used for visualisation similarly to t-SNE, but also for general non-linear 
+                 dimension reduction."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('scikit-learn', '0.24.2'),
+    ('numba', '0.53.1'),
+    ('LLVM', '11.1.0'),
+    ('tqdm', '4.61.1'),
```

unless `umap-learn` has a strong requirement for `tqdm<4.61.2`, it would be good to match the "default" version used by most other easyconfigs in this toolchain generation
```suggestion
    ('tqdm', '4.61.2'),
```